src/hotspot/share/runtime/signature.hpp

```diff
@@ -579,15 +579,16 @@ class SigEntry {
  public:
   BasicType _bt;      // Basic type of the argument
   int _offset;        // Offset of the field in its value class holder for scalarized arguments (-1 otherwise). Used for packing and unpacking.
-  float _sort_offset; // Offset used for sorting
   Symbol* _symbol;    // Symbol for printing
+  bool _null_marker;   // Is it a null marker? For printing
 
   SigEntry()
-    : _bt(T_ILLEGAL), _offset(-1), _sort_offset(-1), _symbol(nullptr) {}
+    : _bt(T_ILLEGAL), _offset(-1), _symbol(nullptr) {}
 
-  SigEntry(BasicType bt, int offset = -1, float sort_offset = -1, Symbol* symbol = nullptr)
-    : _bt(bt), _offset(offset), _sort_offset(sort_offset), _symbol(symbol) {}
+  SigEntry(BasicType bt, int offset, Symbol* symbol, bool null_marker)
+    : _bt(bt), _offset(offset), _symbol(symbol), _null_marker(null_marker) {}
 
+#if 0
```

Comment:
Why do you wrap it with `#if 0` instead of removing this function?

Reply:
It's a leftover that survived the clean up, it should have been removed. I was trying to remove some sorting by removing the order function, and see if it's used somewhere legitimate, but this way it was still easily accessible. Anyway, it should have been, and will be, removed.